lightningd/options.c

```diff
@@ -481,6 +488,9 @@ static const struct config mainnet_config = {
 
 	/* Mainnet should have more stable fees */
 	.ignore_fee_limits = false,
+
+	/* Do not rescan */
+	.rescan = 15,
```

Comment:
This will rescan 15 blocks before the current block height? The comment seems not matching? (should it not be 0 if we just continue scanning at where we left off?)

Also, I remember we had an issue before that our block height would keep getting pushed back if we kept getting restarted, would that behavior occur again?

Reply:
Good catch, I had 0 in there initially and then I remembered that we don't handle reorgs below our in-memory chain all too well, so I made it 15 blocks. If we process less than 15 blocks before crashing, then yes, this could end up walking backwards, we could add the lower bound at the segwit activation back in, but that's also painful.

Reply:
An alternative could be to use `last_processed_block` and not decrement that if it is above our current blockheight. That would make sure that our offset if computed from a fixed point, rather than risking sliding back in time.

Reply:
Comment is still `/* Do not rescan */` even though default is to rescan by 15 blocks back?